wdl-gauntlet/src/lib.rs

```diff
@@ -335,10 +345,20 @@ pub async fn gauntlet(args: Args) -> Result<()> {
             continue;
         }
 
-        for message in messages {
+        let hash = config
+            .inner()
+            .repositories()
+            .get(identifier.repository())
+            .unwrap()
+            .commit_hash()
+            .clone()
```

Comment:
See previous comment, but if we take the hash by reference, we shouldn't need to clone it here or below in the loop.